src/storage/info.h

```diff
@@ -6,6 +6,25 @@ Storage Info
 
 #include <sys/types.h>
 
+/***********************************************************************************************************************************
+Specify the level of information required when calling functions that return StorageInfo
+***********************************************************************************************************************************/
+typedef enum
+{
+    // The info type is determined by driver capabilities.  This mimics the prior behavior where drivers would always return as
+    // much information as they could.
+    storageInfoLevelDefault = 0,
+
+    // Only test for existence.  All drivers support this type.
+    storageInfoLevelExists,
```

Comment:
storageInfoLevelExists is only applicable to *InfoList function vs *Info function which makes this confusing. Why is it not implemented in the *Info function?